jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline {
         stage('Promote Archives') {
             agent {
                 docker {
-                    label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+                    label 'Jenkins-Agent-AL2023-X64-C54xlarge-Docker-Host'
```

Comment:
Let's not change for any non-distribution workflows yet! It would be difficult to test as these are not exercised regularly.

Reply:
Hi,

I think it is ok to proceed with this change as it is simply running a docker container on the host.
Plus it has been there since weeks ago and these servers are stable.

This is a two-way door and I would suggest we go with the change than not so we can actively test these changes from now on. If it failed on that workflow we can always change back.

Thanks.